install_wrapper.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
```

Comment:
Is there a reason why it's not statically done via `simple` buildsystem?

Reply:
this is a wrapper for the blockchain data directory